plc4j/api/src/main/java/org/apache/plc4x/java/api/PlcDriver.java

```diff
@@ -24,6 +24,10 @@ Licensed to the Apache Software Foundation (ASF) under one
 
 /**
  * General interface defining the minimal methods required for adding a new type of driver to the PLC4J system.
+ *
+ * <b>Note that each driver has to add a service file called org.apache.plc4x.java.PlcDriver to
+ * src/main/resources/META-INF which contains the fully qualified classname in order to get loaded
+ * by the {@link java.sql.DriverManager} instances.</b>
```

Comment:
`java.sql.DriverManager` shouldn't that be the `org.apache.plc4x.java.PlcDriverManager`?